src/Persistence/Models/Control.cs

```diff
@@ -113,8 +122,18 @@ internal void BeforeSerialize()
     {
         HideNestedTemplates();
 
+        Comparison<Control> zIndexComparison = (Control c1, Control c2) => c2.ZIndex.CompareTo(c1.ZIndex);
+
+        // For group containers, ZIndex is the reverse order of what is shown in the tree view
+        // Handle that here to ensure we match what is expected
+        if (Template.Name == "GroupContainer")
```

Comment:
Should this constant live somewhere?

Reply:
https://github.com/microsoft/PowerApps-Tooling/blob/master/src/Persistence/Templates/BuiltInTemplates.cs maybe?